tests/test_docs.py

```diff
@@ -12,7 +12,8 @@
 import pytest
 from pytest_examples import CodeExample, EvalExample, find_examples
 
-index_main = None
+INDEX_MAIN = None
+DOCS_ROOT = Path(__file__).parent.parent / 'docs'
```

Comment:
Using this instead of just `'docs'` as the root in the tests makes it work when I run using the PyCharm pytest runner (which apparently doesn't run from the project root?).

I would very much appreciate if we could make this change as the pycharm pytest runner makes it _MUCH_ easier to browse the failures than just looking at console feedback.